packaging/ubuntu-16.04/tests/integrationtests

```diff
@@ -46,7 +46,7 @@ systemctl daemon-reload
 systemctl restart snapd
 
 # Spread will only buid with recent go
-snap install --classic go
+snap install --classic --channel 1.17/stable go
```

Comment:
There's some code elsewhere which does:
```
goinstall() {
    pkg="$1"
    # go1.18+ will no longer build/install packages. Here "go install"
    # must be used but it will only fetch remote packages if the @latest
    # (or similar syntax is used). Instead of checking the version we
    # check if the "go install" help mentions this new feature.
    if go help install | grep -q @latest; then
        go install "${pkg}"@latest
    else
        go get -u "${pkg}"
    fi
}
```
In which case we could just continue following the latest go release.